service/routes.py

```diff
@@ -43,6 +43,29 @@ def index():
 #  R E S T   A P I   E N D P O I N T S
 ######################################################################
 
+@app.route("/promotions/create", methods=["POST"])
```

Comment:
I think the endpoint does not adhere to restful API, as it includes an action term in the URL, but it follows the description of the [story](https://app.zenhub.com/workspaces/promotions-su24-001-666234565dda150276ad19c5/issues/gh/csci-ga-2820-su24-001/promotions/3). Should we consider revising this to align more closely with RESTful best practices?

Reply:
Sure, I'll rename it to **addPromotion**

Reply:
I would recommend using @app.route('/promotions', methods=['POST']) for this endpoint. It seems to align more closely with RESTful principles by avoiding action-specific terms in the URL. What do you think?

Reply:
Sure, that's what I did